src/app/BUILD.gn

```diff
@@ -22,6 +22,18 @@ declare_args() {
   # Enable strict schema checks.
   chip_enable_schema_check =
       is_debug && (current_os == "linux" || current_os == "mac")
+
+  # Logging verbosity control for Access Control implementation
+  #
+  # If set to > 0, it is desired to get additional logging on all
+  # access control checks for better debugging ability.
+  #
+  # If set to > 1, it is desired to log every single check
```

Comment:
Are these descriptions backwards? I.e. >0 means log every check, >1 means additional logging for the checks?

Reply:
>0 means log every check once, >1 is log even more per check